pkgs/development/python-modules/ansible-builder/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, python3Packages
+, podman
+, fetchPypi
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "ansible-builder";
+  version = "3.0.1";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-rxyhgj9Cad751tPAptCTLCtXQLUXaRYv39bkoFzzjOk=";
+  };
+
+  buildInputs = with python3Packages; [
+    setuptools
+    setuptools-scm
+    bindep
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    podman
+    jsonschema
+    requirements-parser
+    pyyaml
+  ];
+
+  patchPhase = ''
+    # the upper limits of setuptools are unnecessary
+    sed -i 's/, <=[0-9.]*//g' pyproject.toml
```

Comment:
I haven't kept up with python packaging but doesn't the relax deps hook support pyproject.toml by now? If so, please use that instead.

Reply:
Sadly it doesn't work with the relax deps hook. But I added a link to the upstream ticket to resolve this issue.